specifications/M1/identity.protocol.base.md

```diff
@@ -0,0 +1,111 @@
+> self-registration, and instead rely on a _**Registration Service**_. Imposing this restriction provides a mechanism
+> for business security (the field of participants can be restricted through a verification process) and greatly
+> simplifies the technical problem at hand.
+
+## 2.1. Decentralization
+
+Decentralization is achieved in the following ways:
+
+- **Self-Issued Identity Tokens** - Each participant is responsible for creating and cryptographically signing its own
+  identity tokens. A central identity provider is therefore not required, eliminating a potential locus of network
+  failure.
+- **Participant VC and Cryptographic Material Management** - Each participant is responsible for storing VCs and
+  presenting them (VPs) in the network. Each participant is also responsible for storing and making its own
+  cryptographic material available to verifiers in the network. This specification defines how VCs are presented and
+  verified without the need for a third-party verification service.
+- **Multiple Trust Anchors** - Participant VCs are signed by a third-party issuer acting as a trust anchor using a
+  cryptographic proof. The identity protocol allows for multiple trust anchors in a dataspace.
+
+# 3. Identities and Identifiers
+
+Each participant MUST have a unique, immutable **_identity_** provided by the `Registration Service` and
+a [Web DID](https://w3c-ccg.github.io/did-method-web/) that it chooses. This relationship is expressed as:
+
+```
+ID  ------ Can resolve to -----> DID
+ ^                                |
+ |                                |
+ |----------Associated with--------                               
+```
+
+This immutable identity is termed the `participant id`.
+
+## 3.1.The Membership VC
+
+Dataspaces which implement the TX identity protocol MUST define a VC that adheres to
+the [Verifiable Credentials Data Model v1.1](https://www.w3.org/TR/vc-data-model/) and cryptographically binds the
+`participant id` to its `Web DID`. This VC is termed the `Membership VC`. The VC verifier's cryptographic material MUST
+be resolvable as a `Web DID`.
+
+> TODO: do we want to define the subject material of the Membership VC or should we leave it dataspace specific?
+
+# 4. Self-Issued Tokens
+
+A self-issued token is defined in
+the [Self-Issued OpenID Provider v2 specification](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html#section-1.1) :
+
+> In the Self-Issued OP case, the ID Token is self-signed with a private key under the user's control, identified by the
+> sub-claim.
+
+A client may obtain a self-issued token using a variety or OAuth grant types. If the OAuth 2.0 Client Credential Grant
+type is used, the client MUST conform
+to [Section 6](#6-using-the-oauth-2-client-credential-grant-to-obtain-access-tokens-from-an-sts).
+
+# 4.1. Self-Issued Token Contents
+
+The self-issued token MUST adhere
```

Comment:
Im not sure you about the Token naming, the VC is based on JsonLD and the VP should be in some part in the Future also JsonLD shouldnt it be named Self Issued Credentials and VP Access Token ?

Reply:
I changed it to Self-Issued ID Token which is used in the SIOP spec.